docs/ref/django-admin.txt

```diff
@@ -1051,6 +1051,21 @@ Mails the email addresses specified in :setting:`ADMINS` using
 
 Starts the Python interactive interpreter.
 
+All models from :setting:`INSTALLED_APPS` are auto-imported to the namespace.
+See the guide on :ref:`customizing the auto-importing behaviour
+<customizing-shell-auto-imports>` to remove or add to the auto-imports.
+
+.. admonition:: Models from different apps with the same name
+
```

Comment:
I don’t think this needs a special box of any kind and it can just be a second paragraph. Admonitions and other boxes are for extra topics, but model name collisions are a natural continuation of the previous paragraph.

```suggestion
```